forge-gui/res/editions/Edge of Eternities.txt

```diff
@@ -4,32 +4,519 @@ Date=2025-08-01
+222 R Mutinous Massacre @Quintin Gleim
+223 R Pinnacle Emissary @Alejandro Pacheco
+224 R Ragost, Deft Gastronaut @Zack Stella
 225 U Sami, Ship's Engineer @Zara Alfonso
+226 M Sami, Wildcat Captain @Kieran Yanner
+227 U Seedship Broodtender @Eric Wilkerson
+228 R Singularity Rupture @Liiga Smilshkalne
+229 R Space-Time Anomaly @Loïc Canavaggia
+230 U Station Monitor @Camille Alquier
+231 R Syr Vondam, Sunstar Exemplar @Ryan Pancoast
+232 U Syr Vondam, the Lucent @Cristi Balanescu
 233 U Tannuk, Memorial Ensign @David Auden Nash
+234 U All-Fates Scroll @Sam Guay
+235 U Bygone Colossus @Maxime Minard
+236 C Chrome Companion @Gray Highsmith
+237 U Dauntless Scrapbot @Alix Branwyn
+238 M Dawnsire, Sunstar Dreadnought @Jaime Jones
+239 M The Dominion Bracelet @Nathaniel Himawan
+240 M The Endstone @Ryan Pancoast
+241 R The Eternity Elevator @Josu Solano
+242 R Extinguisher Battleship @Danny Schwartz
+243 C Nutrient Block @Francisco Miyara
+244 C Pinnacle Kill-Ship @Alexandre Honoré
+245 U Survey Mechan @Johann Bodin
+246 C Thaumaton Torpedo @Madeline Boni
+247 R Thrumming Hivepool @Rob Rey
+248 U Virulent Silencer @Kenn Yap
+249 C Wurmwall Sweeper @Hardy Fowler
+250 M Adagia, Windswept Bastion @Adam Paquette
 251 R Breeding Pool @Constantin Marin
+252 C Command Bridge @Constantin Marin
+253 M Evendo, Waking Haven @Adam Paquette
 254 R Godless Shrine @Rob Rey
+255 M Kavaron, Memorial World @Adam Paquette
 256 R Sacred Foundry @Titus Lunter
+257 R Secluded Starforge @Chris Rahn
 258 R Stomping Ground @Bruce Brenneise
+259 M Susur Secundi, Void Altar @Adam Paquette
+260 M Uthros, Titanic Godcore @Adam Paquette
 261 R Watery Grave @Sergey Glushakov
 262 L Plains @Adam Paquette
 263 L Island @Adam Paquette
 264 L Swamp @Adam Paquette
 265 L Mountain @Adam Paquette
 266 L Forest @Adam Paquette
-287 M Tezzeret, Cruel Captain @Magali Villeneuve
-323 R The Seriema @Sergey Glushakov
-360 M Sothera, the Supervoid @Mateusz Urbanowicz
+267 L Plains @Alayna Danner
+268 L Plains @Sergey Glushakov
+269 L Island @Sergey Glushakov
+270 L Island @Liiga Smilshkalne
+271 L Swamp @Sergey Glushakov
+272 L Swamp @Liiga Smilshkalne
+273 L Mountain @Sergey Glushakov
+274 L Mountain @Julian Kok Joon Wen
+275 L Forest @Sergey Glushakov
+276 L Forest @Julian Kok Joon Wen
+
+[normal lands]
+267 L Plains @Alayna Danner
+268 L Plains @Sergey Glushakov
+269 L Island @Sergey Glushakov
+270 L Island @Liiga Smilshkalne
+271 L Swamp @Sergey Glushakov
+272 L Swamp @Liiga Smilshkalne
+273 L Mountain @Sergey Glushakov
+274 L Mountain @Julian Kok Joon Wen
+275 L Forest @Sergey Glushakov
+276 L Forest @Julian Kok Joon Wen
+
+[borderless lands]
+262 L Plains @Adam Paquette
+263 L Island @Adam Paquette
+264 L Swamp @Adam Paquette
+265 L Mountain @Adam Paquette
+266 L Forest @Adam Paquette
 367 L Plains @Adam Paquette
 368 L Island @Adam Paquette
 369 L Swamp @Adam Paquette
 370 L Mountain @Adam Paquette
 371 L Forest @Adam Paquette
+
+
+[viewport]
```

Comment:
These section names need to match what we can support otherwise they don't work. If you use the latest updates from the edition files they have been grouped to what Forge knows

Reply:
So, I see that these sections are actually defined here: `forge-core/src/main/java/forge/card/CardEdition.java`, So fine, I can take what's in master, however the issue is the distribution of rarities in the borderless section is different from the full draft set. This means It'll be impossible to achieve the stated probabilities here: https://mtgscribe.com/2025/07/16/edge-of-eternities-play-booster-fact-sheet/

I can change all the surreal, triumphant, and viewport sections to references to borderless, but the card frequencies will be incorrect.